test/lib/sql.js

```diff
@@ -0,0 +1,203 @@
+
+const { expect } = require('chai');
+const SQL = require('../../lib/sql');
+const TABLE_NAME = 'test_table';
+
+describe('SQL', () => {
+	describe('Testing find', () => {
+		const expectedQuery = 'select * from `test_table` where `id` = 1';
+		it('Should find by id if parameter is {id: 1}', () => {
+			let error, query;
+			try {
```

Comment:
I think , `try-catch` blocks in tests and having `expect(error).to.be.undefined` are redundant, since if there is an uncaught error, the test will fail anyway.

Reply:
makes sense, I will postpone the refactoring